dev/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  pg:
+    image: postgres:9.3
+
+  conjur_5:
+    image: cyberark/conjur
+    command: server -a cucumber
+    environment:
+      DATABASE_URL: postgres://postgres@pg/postgres
+      CONJUR_DATA_KEY: 'WMfApcDBtocRWV+ZSUP3Tjr5XNU+Z2FdBb6BEezejIs='
+    volumes:
+      - authn_local_5:/run/authn-local
+    depends_on:
+      - pg
+
+  conjur_4:
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:4.9-stable
+    security_opt:
+      - seccomp:unconfined
+    volumes:
+      - ../features_v4/support/policy.yml:/etc/policy.yml
+      - authn_local_4:/run/authn-local
+
+  gem:
+    build:
+      context: ../
+      dockerfile: dev/Dockerfile.dev
+    entrypoint: sleep
+    command: infinity
+    environment:
+      CONJUR_APPLIANCE_URL: http://conjur_5
+      CONJUR_VERSION: 5
+      CONJUR_ACCOUNT: cucumber
+      RESTCLIENT_LOG: stdout
+    links:
+      - conjur_5:conjur_5
+      - conjur_4:conjur_4
+    volumes:
+      - ..:/src/conjur-api
+      - authn_local_4:/run/authn-local
```

Comment:
I needed an additional volume to run the cukes:
```
- authn_local_5:/run/authn-local-5
```
(see the `tester_5` in `docker-compose.yaml` in the project root)